include/fi_util.h

```diff
@@ -192,6 +192,7 @@ struct util_domain {
 	int			mr_mode;
 	uint32_t		addr_format;
 	enum fi_av_type		av_type;
+	struct ofi_mr_map	mr_map;
```

Comment:
As part of this series, can you add a comment to rxd.h to use the util_domain mr_map in place of its own definition.